package.json

```diff
@@ -24,6 +24,7 @@
     "test:try-one": "yarn workspace ember-data test:try-one",
     "test:docs": "yarn workspace ember-data test:docs",
     "test:encapsulation": "yarn workspace serializer-encapsulation-test-app test",
+    "test:adapter-encapsulation": "yarn workspace adapter-encapsulation-test-app test",
```

Comment:
I think instead we can change the `test:encapsulation` command to be 

`lerna run --scope="*-encapsulation-test-app" test`

https://github.com/lerna/lerna/tree/master/commands/run